twisted/spread/flavors.py

```diff
@@ -16,33 +16,35 @@
 @author: Glyph Lefkowitz
 """
 
+from __future__ import absolute_import, division
+
 # NOTE: this module should NOT import pb; it is supposed to be a module which
 # abstractly defines remotely accessible types.  Many of these types expect to
 # be serialized by Jelly, but they ought to be accessible through other
 # mechanisms (like XMLRPC)
 
-# system imports
 import sys
+
 from zope.interface import implementer, Interface
 
-# twisted imports
 from twisted.python import log, reflect
-
-# sibling imports
-from .jelly import setUnjellyableForClass, setUnjellyableForClassTree, setUnjellyableFactoryForClass, unjellyableRegistry
-from .jelly import Jellyable, Unjellyable, _newDummyLike
-from .jelly import setInstanceState, getInstanceState
+from twisted.python.compat import _PY3, unicode, comparable, cmp
+from .jelly import (
```

Comment:
are relative import a thing we want to use in Twisted?

just trying to understand the coding standard